### ISSUE TITLE
panic: runtime error: slice bounds out of range [33:21]

### ISSUE THREAD

**Issue by andsky** [gohugoio/hugo#11295]
### What version of Hugo are you using (`hugo version`)?
<pre>
hugo v0.115.4-dc9524521270f81d1c038ebbb200f0cfa3427cc5+extended linux/amd64 BuildDate=2023-07-20T06:49:57Z VendorInfo=gohugoio
</pre>

### Does this issue reproduce with the latest release?
Yes.

new content
```
<!--[if gte mso 9]与<![endif]-->
```

###Stacktrace
```go
goroutine 23 [running]:
github.com/tdewolff/minify/v2/html.(*Minifier).Minify(0xc000788548, 0xc0005a35c0, {0x28abd20, 0xc0028847e0}, {0x28abce0, 0xc002914180}, 0x0?)
	/root/project/gomodcache/github.com/tdewolff/minify/v2@v2.12.4/html/html.go:99 +0x3d6d
github.com/tdewolff/minify/v2.MinifierFunc.Minify(...)
	/root/project/gomodcache/github.com/tdewolff/minify/v2@v2.12.4/minify.go:41
github.com/gohugoio/hugo/minifiers.Client.Transformer.func1({0x28b88f8?, 0xc003d8b820?})
	/root/project/hugo/minifiers/minifiers.go:51 +0xa3
github.com/gohugoio/hugo/transform.(*Chain).Apply(0xc001f056b0, {0x28abd20, 0xc002ab2240}, {0x28abce0, 0xc002884600})
	/root/project/hugo/transform/chain.go:108 +0x3af
github.com/gohugoio/hugo/publisher.DestinationPublisher.Publish({{_, _}, {_, _}, _}, {{0x28abce0, 0xc002884600}, {{0x2217e46, 0x4}, {{0x2219ade, ...}, ...}, ...}, ...})
	/root/project/hugo/publisher/publisher.go:107 +0x1a5
github.com/gohugoio/hugo/hugolib.(*Site).renderAndWritePage(0xc00072dd40, 0xc0004866f0, {0xc002848ed0, 0x2f}, {0xc001d2c2e8, 0x15}, 0xc000eac270, {0x28b88a8, 0xc0005aa3c0})
	/root/project/hugo/hugolib/site.go:1784 +0x7f5
github.com/gohugoio/hugo/hugolib.pageRenderer(0x0?, 0xc00072dd40, 0x0?, 0x0?, 0x0?)
	/root/project/hugo/hugolib/site_render.go:138 +0x287
created by github.com/gohugoio/hugo/hugolib.(*Site).renderPages
	/root/project/hugo/hugolib/site_render.go:74 +0x108
```

**Comment by bep** [comment]
Thanks, this seem to be a upstream issue. I would appreciate if you could create an issue at 

https://github.com/tdewolff/minify/issues

And point back to this issue. We're using v2.12.7 of minfiy, which I see is the latest.

**Comment by jmooring** [comment]
<https://github.com/tdewolff/minify/issues/596>

**Comment by jmooring** [comment]
This was resolved in v0.120.0 with https://github.com/gohugoio/hugo/pull/11535
